04_numeronym/marchitect/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+var out io.Writer = os.Stdout
+
+func numeronyms(vals ...string) []string {
+	result := make([]string, 0)
+	for _, val := range vals {
+		numeronym := ""
+		length := len([]rune(val))
```

Comment:
It's good you took into account Unicode characters. Initially when I wrote my function it worked for normal english alphabet characters. But once I put something like an emoji it gave me weird results until I converted them to a rune as well